web/server/vue-cli/src/components/Product/ProductConfigForm.vue

```diff
@@ -39,6 +39,26 @@
       @input="productConfig.runLimit = $event || null"
     />
 
+    <v-row
+      class="ma-0"
+    >
+      <v-text-field
+        v-if="isSuperUser"
+        :value="productConfig.reportLimit"
+        type="number"
+        label="Report limit"
+        name="report-limit"
+        prepend-icon="mdi-close-octagon"
+        :rules="rules.runLimit"
+        @input="productConfig.reportLimit = $event || null"
+      />
+
+      <tooltip-help-icon>
+        Maximum number of reports allowed to store in one go,
```

Comment:
One go? You mean that this is the maximum size of _any one store action_, not the maximum of the _sum of ongoing store actions_. I know I am nitpicking, but lets just make this a little more precise.

Reply:
Clarified the tooltip